dz19/src/components/Header/App.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from './App.module.css'
+
+export default function Header() {
+  return (
+    <header>
+      <nav className={styles.nav}>
+        <ul className={styles.ul}>
+          <li ><a href="home">Главная</a></li>
```

Comment:
```suggestion
          <li><a href="/">Главная</a></li>
```

ссылки обычно начинаются с `/` (корень сайта)